### ISSUE TITLE
webdriver_manager not getting the latest driver

### ISSUE THREAD

**Issue by RamiroMM** [SergeyPirogov/webdriver_manager#116]
My chrome got updated to v83. Still, the manager is looking to run v81 chromedriver.

I've used the manager for long enough, and it's the first time it's not getting the latest version.

![image](https://user-images.githubusercontent.com/26583428/82582994-73c5ed80-9b58-11ea-8438-332d3038fc0a.png)

And it's weird cause this link shows that 83 is the latest one:
http://chromedriver.storage.googleapis.com/LATEST_RELEASE

But somehow it's sending a True in is_latest.

Am I missing a step?

**Comment by SergeyPirogov** [comment]
Need to investigate

**Comment by SergeyPirogov** [comment]
CI tests works fine with chrome 83

**Comment by RamiroMM** [comment]
Mmm that's weird.. let me try again and see what happens

**Comment by SergeyPirogov** [comment]
If it still broken, please provide chrome driver version

**Comment by RamiroMM** [comment]
Yeah, it's still looking for the previous version. But providing the specific driver version works fine. That'd do :)

Thanks for the quick response!

**Comment by SergeDmitriev** [comment]
Good day! I faced the same problem on my CI and local system. Removing file ".../.wdm/drivers.json" helps in both cases:)

**Comment by RamiroMM** [comment]
@SergeDmitriev thanks for the comments! I did try that as well... it didn't work for me :c 
But specifying the version worked. We now have 2 workarounds :)

**Comment by heesangkim** [comment]
> Good day! I faced the same problem on my CI and local system. Removing file ".../.wdm/drivers.json" helps in both cases:)

Thank you, this solution worked for me.

**Comment by SergeyPirogov** [comment]
Cache valid for 1 day. We can try to add - - force key
